ci/lint/check-documentation-style.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+set -euxo pipefail
+
+wget https://github.com/errata-ai/vale/releases/download/v2.28.0/vale_2.28.0_Linux_64-bit.tar.gz
+VALE_BIN=$(mktemp -d)
```

Comment:
you would want to

1. `mktemp -d`
2. download the tarball into the temp dir
3. extract the binary into the temp dir
4. run vale
5. remove the tempdir

general principle is that don't write anything outside the tempdir onto the filesystem.

you can also `curl | tar -xvzf -C ` in one command, since you have `pipefail` set.